mono/pal/Makefile.am

```diff
@@ -0,0 +1,18 @@
+
+COREFX_ROOT=$(abs_top_srcdir)/external/corefx
+
+all-local:
+	$(COREFX_ROOT)/src/Native/build-native.sh $(COREFX_PAL_ARCH) Debug $(COREFX_PAL_OS) \
+		cmakeargs -DCLR_DISABLE_SYSTEM_IO_COMPRESSION_NATIVE:BOOL=1 \
+		cmakeargs -DCLR_DISABLE_SYSTEM_NET_HTTP_NATIVE:BOOL=1 \
+		cmakeargs -DCLR_DISABLE_SYSTEM_NET_SECURITY_NATIVE:BOOL=1 \
+		cmakeargs -DCLR_DISABLE_SYSTEM_SECURITY_CRYPTOGRAPHY_NATIVE:BOOL=1 \
+		cmakeargs -DCLR_DISABLE_SYSTEM_SECURITY_CRYPTOGRAPHY_NATIVE_APPLE:BOOL=1
+	ln -fs $(COREFX_ROOT)/bin/$(COREFX_PAL_OS).$(COREFX_PAL_ARCH).Debug/native/System.Native$(libsuffix) System.Native$(libsuffix)
+
+clean-local:
+	-rm -rf System.Native$(libsuffix)
```

Comment:
This needs an additional line:
`cd $(COREFX_ROOT) && ./clean.sh -all`
(Running script without cd'ing will perform git clean on wrong dir; running without `-all` requires .NET CLI to be installed)